man/systemd.service.xml

```diff
@@ -77,6 +77,19 @@
     with SysV</ulink> document.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Service Templates</title>
+
+    <para>It is possible for Systemd services to take a single argument via the
+    <literal><replaceable>service</replaceable>@<replaceable>argument</replaceable></literal>
+    syntax. Such services are called "instantiated" or "template" services. An
+    example could be a <filename>dhcpcd@</filename> service which takes a network
+    interface as a parameter. Within the service file, this parameter or
+    "instance name" can be accessed with %-specifiers. See
```

Comment:
The `%` should probably be enclosed in `<literal>`?

Reply:
I don't think so. For me the percent sign is part of the name.